### PR TITLE
feat: add navbar rough template

### PR DESCRIPTION
**Pull request by DistractionBoy**
adding an initial template. need to componentize and only show what's ready. 

work in progress

### CODE REVIEW

**Comment by DistractionBoy** [comment]
https://tailwindcss.com/docs/dark-mode

File: projects/marketing-site/components/DarkNavbar/DarkNavbar.tsx [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+          <div className="mx-auto max-w-7xl px-2 sm:px-4 lg:px-8">
+            <div className="relative flex h-16 items-center justify-between">
+              <div className="flex items-center px-2 lg:px-0">
+                <div className="flex-shrink-0">
+                  <Image
+                    className="block h-8 w-auto lg:hidden"
+                    src={logoSVG}
+                    height={32}
+                    width={32}
+                    alt="Your Company"
```
Comment (Millsky) [inline projects/marketing-site/components/DarkNavbar/DarkNavbar.tsx:39]: This should be `HyperDrive` or something like it.

File: projects/marketing-site/components/DarkNavbar/DarkNavbar.tsx [hunk 2]
```diff
@@ -0,0 +1,281 @@
...
+                  />
+                </div>
+                <div className="hidden lg:ml-6 lg:block">
+                  <div className="flex space-x-4">
+                    {/* Current: "bg-gray-900 text-white", Default: "text-gray-300 hover:bg-gray-700 hover:text-white" */}
+                    <a
+                      href="#"
+                      className="rounded-md bg-gray-900 px-3 py-2 text-sm font-medium text-white"
+                    >
+                      Dashboard
```
Comment (Millsky) [inline projects/marketing-site/components/DarkNavbar/DarkNavbar.tsx:56]: Maybe parameterize this so we can do translations?

File: projects/marketing-site/components/DarkNavbar/DarkNavbar.tsx [hunk 3]
```diff
@@ -0,0 +1,281 @@
...
+                      className="block w-full rounded-md border border-transparent bg-gray-700 py-2 pl-10 pr-3 leading-5 text-gray-300 placeholder-gray-400 focus:border-white focus:bg-white focus:text-gray-900 focus:outline-none focus:ring-white sm:text-sm"
+                      placeholder="Search"
+                      type="search"
+                    />
+                  </div>
+                </div>
+              </div>
+              <div className="flex lg:hidden">
+                {/* Mobile menu button */}
+                <Disclosure.Button className="inline-flex items-center justify-center rounded-md p-2 text-gray-400 hover:bg-gray-700 hover:text-white focus:outline-none focus:ring-2 focus:ring-inset focus:ring-white">
```
Comment (Millsky) [inline projects/marketing-site/components/DarkNavbar/DarkNavbar.tsx:103]: Can we make the mobile menu a separate component?

**Review by Millsky — COMMENTED** [review]
Some small nits, for the overview could you post a Screen shot too so when someone looks at history it's clear what the component looked like @ this stage?

**Pull request merged**